spec/models/cinema_hall_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe CinemaHall, type: :model do
+
+  it 'valid with correct info' do
```

Comment:
Remember that description in the `it` block should contain information about what this particular test is testing. I know that this file is under development, but it's an early stage of code review :)